tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "sourceMap": true,
     "experimentalDecorators": true,
     "outDir": "./dist",
-    "lib": ["es2015"]
+    "lib": ["es2015", "dom"]
```

Comment:
Added `dom` here because the demo file uses `console.log` and I didn't want to exclude the demo from transpiling.